WeatherApp/WeatherApp/HomeScreen/HomeScreenView.swift

```diff
@@ -19,6 +19,10 @@ struct HomeScreenView: View {
                     .padding()
                 Text("Wind Speed: \(String(format: "%.1f", weather.wind.speed)) m/s")
                     .padding()
+                Text("Sunrise: \(homeScreenViewModel.formatTimeFromUnix(weather.sys.sunrise, timeZoneOffset: 3600))")
+                    .padding()
```

Comment:
Since `HomeScreenView` and `HomeScreenViewModel` are not used anymore, its safe to delete them